### DISCUSSION TITLE
Manual reading (See also Discussion #1410)

### DISCUSSION THREAD

**Discussion by RasputinBerry** [jomjol/AI-on-the-edge-device#1937] [General] [answered]
### The Feature

I would be happy if a reading could be forced in a targeted manner. For example, I start the reading before I start the washing machine and again at the end of the wash cycle to determine the water consumption.

**Comment by caco3** [comment, marked as answer]
This is now possible, see https://jomjol.github.io/AI-on-the-edge-device-docs/REST-API/#flow_start

**Reply by RasputinBerry** [reply]
OK ... but "192.168.178.xx/flow_start" returns this error... I'm probably doing something wrong there, right ? (Release: v13.0.8 )
![grafik](https://user-images.githubusercontent.com/56359831/215323887-7c6ff9b7-10a4-4acb-bb33-871a44373d4d.png)

**Reply by caco3** [reply]
yes, right. 
it is coming in the v14 release.
you can test it already: https://github.com/jomjol/AI-on-the-edge-device/releases/tag/v14.0.0-RC1

**Reply by RasputinBerry** [reply]
Great... I'm waiting...

**Reply by RasputinBerry** [reply]
... I installed stable version 14. No matter when I run the command ... always the same output ... 

![grafik](https://user-images.githubusercontent.com/56359831/216693687-c9a79dab-0745-4201-8e68-d9577adced69.png)
The current value should be here, right?  Did I misunderstand something?

**Reply by caco3** [reply]
please check if the log shows some reasons why it nod not start

**Reply by RasputinBerry** [reply]
... ... no errors or similar in the log ...

**Reply by Slider0007** [reply]
> ... I installed stable version 14. No matter when I run the command ... always the same output ...
> 
> ![grafik](https://user-images.githubusercontent.com/56359831/216693687-c9a79dab-0745-4201-8e68-d9577adced69.png) The current value should be here, right? Did I misunderstand something?

This command only starts the flow again if it is in the state "flow finished", otherwiese it's doing nothing bercause processing is already ongoing. Just go back to main screen and check if the flow got started. After a while (1-2min) you will get the new result of the new started round. The result can be called with a different API request, e.g. http://192.168.1.1/json. Please have a look to the https://jomjol.github.io/AI-on-the-edge-device-docs/REST-API/

**Reply by RasputinBerry** [reply]
... OK, my "AutoTimer interval" is set to 60 minutes ... so the "flow" should also finished sometime ...
I would expect a result like "http://192.168.178.88/cpu_temperature", then I get e.g. "42" back,  without another call to "json" ...

**Reply by Slider0007** [reply]
As I mentioned "flow_start" just triggers the start of a flow (this is an input to the system). After triggering, just use your back button in explorer to go back to main screen and you should see that a new flow was started. After same time the result can be called using e.g. \json.
The /cpu_temperature is an output like e.g the /json, therefore you get back results on the screen.

**Reply by RasputinBerry** [reply]
OK, it works the way you described it, but ...
- I think the "The flow is going to..." output is a bit confusing. It looks more like an "error message".  You have to know (or guess) that there is no direct value or you have to go back.
- It would be better to output the value directly (without going through JSON) or to switch directly to the "Overview" page. 
- Or a "flow start button" in the WebUI

**Anyway, the project is great !!!**

**Reply by Slider0007** [reply]
Usually the API is triggered remotely by another system, therefore no need to handle the visual feedback.
A button on UI would potentially be possible, but actually not planned right now... You could add a feature request for this.

**Comment by RasputinBerry** [comment]
... for my example from the beginning of this discussion it would be easier if the value was output directly or reachable via the WebUI ... I'm writing a feature request => https://github.com/jomjol/AI-on-the-edge-device/issues/1971
